ldap_auth_provider.py

```diff
@@ -88,7 +88,7 @@ def check_password(self, user_id, password):
         localpart = user_id.split(":", 1)[0][1:]
 
         try:
-            server = ldap3.Server(self.ldap_uri, get_info=None)
+            server = ldap3.ServerPool([ldap3.Server(uri, get_info=None) for uri in self.ldap_uri])
```

Comment:
If `self.ldap_uri` is still a string you'll now add each character as a server. Very much a breaking change.

Reply:
Absolutely! [Fixed](https://github.com/matrix-org/matrix-synapse-ldap3/pull/85/commits/e0553ca21294950ebdded95e8003aa01fde7f498).